doc/index.rst

```diff
@@ -35,8 +35,21 @@
     reference/*
 
 
-Project Summary
-===============
+What is QGreenland?
+===================
+
+QGreenland is a free and open-source Greenland-focused environment for data
+analysis and viewing, powered by QGIS. QGreenland is delivered in two ways:
```

Comment:
Couple of suggestions: 

* Maybe add "GIS" to the first setnence "QGreenland is a free and open-source Greenland-focused GIS environment"? Or "geospatial environment"?
* Make "QGIS" a clicky link :)

Reply:
d5bdad36